### PR TITLE
Dockerfile: install android sdk/ndk

### PR DESCRIPTION
**Pull request by chunseoklee**
This commit prepares android aar build

ONE-DCO-1.0-Signed-off-by: Chunseok Lee <chunseok.lee@samsung.com>

### CODE REVIEW

**Comment by chunseoklee** [comment]
docker image built on this commit succeed to generate one.aar

**Comment by chunseoklee** [comment]
Size of result docker image : 5.53 GB

**Comment by chunseoklee** [comment]
our internal app build succeed on this docker image, too.

**Review by hseok-oh — DISMISSED** [review]

File: infra/docker/Dockerfile.1804 [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# Copyright 2016-2020 Jing Li
```
Comment (seanshpark) [inline infra/docker/Dockerfile.1804:1]: If this file began from ours then I think we should add our copyright also

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Review by hseok-oh — APPROVED** [review]

**Pull request merged**